app/assets/stylesheets/modules/sul-icons.scss

```diff
@@ -1,10 +1,6 @@
 .rss-icon {
-  // replace this extend when we have SUL Styles available to us
-  @extend .fa;
-  @extend .fa-lg;
-  @extend .fa-rss-square;
-
-  color: $sul-rss-icon-color;
+  font-size: 1.1rem;
```

Comment:
Probably a bit nitpicky but right now the height on the rss icon is 16px. If you change this to 1rem it is exactly what we have right now.

Reply:
It looks to me like it's overflowing the 16px box, which is why I made it bigger.
<img width="455" alt="Screenshot 2025-05-12 at 5 21 05 PM" src="https://github.com/user-attachments/assets/de69baa7-f120-4fcc-806a-d5d4a87d1fc8" />

Reply:
Isn't the 16 the width? and 14 the height?